mongo/integration/initial_dns_seedlist_discovery_test.go

```diff
@@ -63,6 +73,24 @@ func runSeedlistDiscoveryDirectory(mt *mtest.T, subdirectory string) {
 	}
 }
 
+// runSeedListDiscoverPingTest will create a new connection using the test URI and attempt to "ping" the server.
+func runSeedListDiscoveryPingTest(mt *mtest.T, clientOpts *options.ClientOptions) {
+	ctx := context.Background()
+
+	client, err := mongo.Connect(ctx, clientOpts)
+	assert.Nil(mt, err, "Connect error: %v", err)
+
+	defer func() { _ = client.Disconnect(ctx) }()
+
+	// Create a context with a timeout to prevent the ping operation from blocking indefinitely.
+	pingCtx, cancel := context.WithTimeout(ctx, 1*time.Second)
+	defer cancel()
+
+	// Ping the server.
+	err = client.Ping(pingCtx, readpref.Nearest())
```

Comment:
Since this is just a ping operation to see if the server is running, I think using the nearest is just the quickest solution. Though I'm sure the performance of any read preference would be negligible in comparison to nearest.